### ISSUE TITLE
Update Drop Shadow effect in UI Design

### ISSUE THREAD

**Issue by maxatic** [getAlby/lightning-browser-extension#2293]
### What I want to implement?

Hello everyone! I investigated a Figma file and noticed that the shadowing was a little inconspicuous - maybe even standard - that Figma gives if you just press the effects button. The shadow effect I want to represent is smoother and more detectable. Additionally, there are some pages where the main color palette of Alby is not shown. That's why we can also add the color of the object and make the shadow more "orange," like in the screenshot below.

### Why we need to implement it?

I believe it will make the process more comprehensive and user-friendly. We could test it through A/B tests if necessary. The whole reason I want to change the shadow effect is that it will make the shadow softer and more in line with Alby's current design.

### How to implement it?

1.     Choose the necessary component that you want to change.
2.     Click on the "Change" button in the "Drop Shadow Effects" panel.
3.     Write down all the values that I presented in the screenshot.

_**Sorry If I did something wrong. That's my first issue here 😄**_ 

![Drop Shadow Issue](https://user-images.githubusercontent.com/68378768/228789562-73de6264-ae18-4011-9d85-74cdaf417887.png)

![Drop Shadow Issue 2](https://user-images.githubusercontent.com/68378768/228789567-16bac9d4-e05c-4d1f-a91a-f8a188b30cec.png)

**Comment by stackingsaunter** [comment]
Seems to me those shadows may be too big/spread for those elements. Did you try to see how would they look like in the screen? Can you share a screenshot?

We have to be careful with that effects. Bigger/smoother shadow implies the elements is on higher level. This may have usage eg. in floating buttons. Those card do not necessarily need to be "high" or have something beneath. You can read about it eg. [here](https://m3.material.io/styles/elevation/applying-elevation) and [here ](https://m2.material.io/design/environment/elevation.html#elevation-in-material-design)

Also keep in mind we're using [shadows from Tailwind CSS](https://tailwindcss.com/docs/drop-shadow)

**Comment by SumantxD** [comment]
@maxatic could you assign me this issue, i would like to work on it.

**Comment by maxatic** [comment]
> Seems to me those shadows may be too big/spread for those elements. Did you try to see how would they look like in the screen? Can you share a screenshot?
> 
> We have to be careful with that effects. Bigger/smoother shadow implies the elements is on higher level. This may have usage eg. in floating buttons. Those card do not necessarily need to be "high" or have something beneath. You can read about it eg. [here](https://m3.material.io/styles/elevation/applying-elevation) and [here ](https://m2.material.io/design/environment/elevation.html#elevation-in-material-design)
> 
> Also keep in mind we're using [shadows from Tailwind CSS](https://tailwindcss.com/docs/drop-shadow)

Hello! Thank you! I got the idea, and will consider Tailwind CSS in this case.

**Comment by HARI-124** [comment]
Is this issue need to be worked on?

**Comment by stackingsaunter** [comment]
@HARI-124 I would say not yet. @maxatic do you still want to play with shadows design?

**Comment by maxatic** [comment]
Yes, I think we can work on shadow effect. Main things here is I think surveys and A/B testing (not only implementing shadow effect itself). I would be happy to work with it right now, but I think I am not capable of it because I broke my right arm and its in a cast. Took a while to write this message)

**Comment by taboca** [comment]
@maxatic Regarding the arm, I recommend you take rest with the fingers too and check if the typing does not cause impact to your recovery. I see that this issue, like many others in the project that touches front-end, offers a chance for us to learn about user experience and design. 

As an example I found insightful the comment from @stackingsaunter https://github.com/getAlby/lightning-browser-extension/issues/2293#issuecomment-1490031818 with the points about shadows. 

Another one I like to remember us, about the use of large shadows, is when these UI elements are inside boxes, such as a small window, and the soft-large shadow tends to hit the border. In other words, large shadows help us to see an element (as if it is lifted) when this element is also far from the limits of the parent element (such as a window). 

I have seen that some of the ui screens of Alby also appear inside the extension pop-up - which has different window limits. Thus, we have to watch out when certain elements are both used in the extension pop-up and also the in-page screens.
